src/Containers.jsx

```diff
@@ -255,6 +255,15 @@ const ContainerActions = ({ container, healthcheck, onAddNotification, version,
         }
     }
 
+    if (container && containerDetail && localImages) {
+        actions.push(
+            <DropdownItem key="clone-container"
+                          onClick={() => copyContainer(container, containerDetail, localImages)}>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1270-20230428-110256-01fd85db-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)